dnsdist 1.1.0beta2 needs -latomic on powerpc

I know this is getting old, but apparently dnsdist 1.1.0beta2 needs to add -latomic to LDFLAGS to build on powerpc. (It already automatically adds it for mipsel.)

```
libtool: link: powerpc-linux-gnu-g++ -fPIE -DPIE -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 --param ssp-buffer-size=4 -fstack-protector -g -O2 "-fdebug-prefix-map=/«PKGBUILDDIR»=." -specs=/usr/share/dpkg/pie-compile.specs -fstack-protector-strong -Wformat -Werror=format-security -pie -pthread -Wl,-z -Wl,relro -Wl,-z -Wl,now -specs=/usr/share/dpkg/pie-link.specs -Wl,-z -Wl,relro -Wl,-z -Wl,now -o dnsdist bpf-filter.o dns.o dnscrypt.o dnsdist.o dnsdist-dynbpf.o dnsdist-cache.o dnsdist-carbon.o dnsdist-console.o dnsdist-dnscrypt.o dnsdist-ecs.o dnsdist-lua.o dnsdist-lua2.o dnsdist-protobuf.o dnsdist-rings.o dnsdist-tcp.o dnsdist-web.o dnslabeltext.o dnsname.o dnsparser.o dnsrulactions.o dnswriter.o ednsoptions.o ednscookies.o ednssubnet.o gettime.o iputils.o misc.o protobuf.o qtype.o remote_logger.o sodcrypto.o statnode.o ext/json11/json11.o dnsmessage.pb.o -pthread  -llua5.2 -ledit -lncurses -lbsd ./ext/yahttp/yahttp/.libs/libyahttp.a -lsodium -lsystemd -lre2 -lprotobuf -lpthread -pthread
/usr/bin/ld: dnsdist.o: undefined reference to symbol '__atomic_fetch_sub_8@@LIBATOMIC_1.0'
//usr/lib/powerpc-linux-gnu/libatomic.so.1: error adding symbols: DSO missing from command line
```

Debian build log here:
https://buildd.debian.org/status/fetch.php?pkg=dnsdist&arch=powerpc&ver=1.1.0%7Ebeta2-1&stamp=1482012656

1.0.0 did not need it.

--- a/m4/pdns_check_os.m4
+++ b/m4/pdns_check_os.m4
@@ -39,6 +39,9 @@ AC_DEFUN([PDNS_CHECK_OS],[
   mips*)
     LDFLAGS="-latomic $LDFLAGS"
     ;;
+  powerpc*)
+    LDFLAGS="-latomic $LDFLAGS"
+    ;;
   esac
 
   AC_SUBST(THREADFLAGS)

